lib/ansible/module_utils/crypto.py

```diff
@@ -63,7 +63,7 @@ def load_privatekey(path, passphrase=None):
         if passphrase:
             privatekey = crypto.load_privatekey(crypto.FILETYPE_PEM,
                                                 open(path, 'rb').read(),
-                                                passphrase)
+                                                passphrase.encode('UTF-8'))
```

Comment:
Should not `to_bytes` used instead of `encode('UTF-8')` (same remark applies to `openssl_privatekey.py`)?